elm-compiler.cabal

```diff
@@ -160,11 +160,13 @@ Executable elm
         CommandLineRouter.hs
 
     other-modules:
-        Elm.Compiler.Version
+        Elm.Compiler.Version,
+        Paths_elm_compiler
 
     Build-depends:
         base >=4.2 && <5,
         directory >= 1.0 && < 2.0,
+        elm-compiler,
```

Comment:
Why do we need to add this? The code does not depend on elm-compiler. It uses Elm.Compiler.Version [here](https://github.com/elm-lang/elm-compiler/blob/master/src/CommandLineRouter.hs#L9), so I get the transitive other-modules addition. It is not clear why that requires the elm-compiler dependency though, or what that would even mean.